.github/workflows/formatting.yml

```diff
@@ -16,5 +16,5 @@ jobs:
           sudo apt-get install --no-install-recommends -qqy clang-format
       - name: Check actual code formatting
         run: |
-          clang-format -i src/*.cpp include/*.hpp
+          git ls-files | grep -v simulator-gui/rlImGui/ | grep -v simulator-gui/imgui/ | grep -e '\.cpp$' -e '\.hpp$' -e '\.c$' -e '\.h$' | xargs clang-format -i
```

Comment:
Having a simple bash script would be cool to have : coping this line when Ci does not pass may be painful in a long term approach.

Reply:
I agree, I got used to this command because I used the history search keybind (Ctrl+R) of Bash to format the files, but a proper script would be nice.